.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [8, 10, lts/*]
```

Comment:
I think we can safely drop 8 and 10 considering they've been off LTS for a long time. https://nodejs.org/en/about/releases/

Reply:
Ok sounds good, want me to do a major release bumping the node version?

Reply:
Merged as is. Let me know how you want that handled.